CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+2.0.0 (In progress)
+===================
+
+New Features
+------------
+
+- Added shims for WebRTC 1.0 APIs. The legacy API shims have been
```

Comment:
We may want to be a little more conservative and just mention the actual methods we added (e.g., the RTCRtpReceiver- and RTCRtpSender-related methods).